mw-webapp/src/component/table/Table.tsx

```diff
@@ -1,40 +1,34 @@
-import {useEffect, useState} from "react";
-import {
-  flexRender,
-  getCoreRowModel,
-  useReactTable,
-} from "@tanstack/react-table";
-import {columns} from "src/component/table/columns";
-import {getDayReports} from "src/dataAccessLogic/getDayReports";
+import {memo} from "react";
+import {flexRender} from "@tanstack/react-table";
 import {DayReport} from "src/model/businessModel/DayReport";
+import {ReportsTableProps} from "src/pages/reportsTable/ReportsTable";
 import styles from "src/component/table/Table.module.scss";
 
 /**
- * Table (need update for split component and logic code)
+ * Table's props
  */
-export const Table = () => {
-  const [data, setData] = useState<DayReport[]>([]);
+interface TableProps<T> {
+  /**
+  * Table's data
+  */
+  data: ReportsTableProps<T>;
```

Comment:
Why data is ReportsTableProps? If it's generic then we can use ant data for create any table/ Image that you will create table with users - ib this case data can't be Named ReportsTableProps

Reply:
Fixed